pkg/workflow/asset/data/schemas/v1/Workflow.json

```diff
@@ -0,0 +1,94 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Workflow",
+  "description": "A project nebula workflow definition",
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "type": "string",
+      "description": "Nebula workflow schema version",
+      "enum": [
+        "v1"
+      ]
+    },
+    "description": {
+      "type": "string",
+      "description": "An optional user-provided workflow description"
+    },
+    "parameters": {
+      "type": "object",
+      "patternProperties": {
+        "(.*?)": {
+          "$ref": "#/definitions/Parameter"
+        }
+      },
+      "additionalProperties": false
+    },
+    "steps": {
+      "type": "array",
+      "description": "List of workflow steps",
+      "items": {
+        "$ref": "#/definitions/Step"
+      }
+    }
+  },
+  "required": [
+    "apiVersion",
```

Comment:
We'll have to make this (and `kind`) optional to support existing workflows. When we finally get to a `v2` we can then make both mandatory.